src/figuresClasses.ts

```diff
@@ -1,11 +1,79 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    private a: number,
+    private b: number,
+    private c: number,
```

Comment:
[CODE KNOWLEDGE] - There's no need to manually assign the constructor parameters to class properties since TypeScript can do this automatically when using access modifiers like 'public' or 'private'. You can remove the assignments inside the constructor body and the class will still have these properties with the passed values.